test/t_cose_crypto_test.h

```diff
@@ -13,6 +14,26 @@
 
 #include <stdint.h>
 
+#if defined(T_COSE_USE_OPENSSL_CRYPTO)
```

Comment:
This header is the interface file for tests against the crypto, not a generic header for all tests.

Also, these #define are just for one test so they should be only in the .c file that has that test.

Thus, all changes made to this file should be removed.

Reply:
Done